modules/core/packet-server/keeper/keeper.go

```diff
@@ -152,8 +152,6 @@ func (k Keeper) RecvPacket(
 		return "", channeltypes.ErrNoOpMsg
 	}
 
-	// create key/value pair for proof verification by appending the ICS24 path to the last element of the counterparty merklepath
-	// TODO: allow for custom prefix
```

Comment:
This todo is already addressed by prefixing being in the counterparty? What does "custom prefix" mean otherwise?